src/components/modal/LoginByEmailModal.vue

```diff
@@ -25,25 +27,30 @@
       />
 
       <v-select
-        :disabled="isLoading"
+        :disabled="isLoading || isValidating"
         :options="availableIdentityServerTypes"
         v-model="currentIdentityServerType"
         label="Identity Server"
         name="currentIdentityServerType"
       />
 
       <v-input
-        v-if="selectedCustomIdentityServerType"
+        v-if="isSelectCustomIdentity"
         id="customIdentityServer"
+        key="custom-identity-server"
         v-model="customIdentityServer"
-        :disabled="isLoading"
+        :disabled="isLoading || isValidating"
```

Comment:
[The same story](https://github.com/endpass/endpass-wallet/pull/231/commits/73a2fae8a0e7eab9caad804a5d39bb74bf411d5d#diff-b795ab9117fc77682004197e264042a6R30)